bench/release.py

```diff
@@ -207,7 +199,7 @@ def create_github_release(owner, repo, tag_name, log, gh_username=None, gh_passw
 			raise Exception, "No credentials"
 		gh_username = github_username
 		gh_password = github_password
-	repo = repo_map[os.path.basename(repo)]
+	repo = os.path.basename(repo)
```

Comment:
The only problem with this assumption is that the repo_name will be same as the folder name. This is problematic in cases where folder_name is say, `frappeclient`, but repo name is `frappe-client`. So, we should add an option for --repo_name for such cases.